### PR TITLE
pkg/visitor

### PR DESCRIPTION
**Pull request by timburks**
Move Get*(), List*(), Fetch*(), Visit(), and Handlers to pkg/visit.

### CODE REVIEW

File: pkg/visitor/fetch_test.go [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+func TestFetch(t *testing.T) {
+	projectID := "fetch-test"
+	project := names.Project{ProjectID: projectID}
+	parent := project.String() + "/locations/global"
+
+	specContents := "hello"
+	artifactContents := "hello"
+
+	ctx := context.Background()
+	adminClient, err := connection.NewAdminClient(ctx)
```
Comment (theganyo) [inline pkg/visitor/fetch_test.go:37]: Do we want to start using `grpctest.SetupRegistry()`?

File: pkg/visitor/visit_test.go [hunk 2]
```diff
@@ -0,0 +1,600 @@
...
+
+	const ProjectCount = 1
+	const ApiCount = 3
+	const VersionCount = 2
+	const DeploymentCount = 2
+	const SpecCount = 2
+	const ArtifactCount = 2
+
+	ctx := context.Background()
+	adminClient, err := connection.NewAdminClient(ctx)
```
Comment (theganyo) [inline pkg/visitor/visit_test.go:51]: grpctest.SetupRegistry?

File: pkg/visitor/fetch_test.go (hunk 1 above)
Comment (timburks) [reply pkg/visitor/fetch_test.go:37]: Yes, let's do some dedicated PRs for that change. I would have used it here but didn't feel that I knew the pattern well enough yet.

File: pkg/visitor/visit_test.go (hunk 2 above)
Comment (timburks) [reply pkg/visitor/visit_test.go:51]: ack, agreed

**Pull request merged**